plugin.js

```diff
@@ -199,18 +708,18 @@
 						{
 							editor.on('selectionChange', function(ev)
 							{
-								var regexp = new RegExp('^' + placeholdersRegex + '$');
 								var currentValue = this.getValue();
 
 								var elements = ev.data.path.elements;
 								for(var i = 0; i < elements.length; i ++)
 								{
 									var element = elements[i];
-									var text = element.$.innerText.toString();
-									if(text !== currentValue && text.match(regexp))
-									{
-										this.setValue(text, placeholdersMap[text].label);
-										return;
+									if (element.$.innerText !== undefined) {

```

Comment:
i honestly forgot about this change...  but i have pushed another commit that changes ```innerText``` to ```textContent``` which I believe is more standard and will work in Firefox.